### ISSUE TITLE
Name Field (Simple, Normal, Extended)

### ISSUE THREAD

**Issue by ghost** [pressoholics/prso-gravityforms-foundation-5#2]
it seems that when you select simple name format it will not display one input field instead of the two? The html seems to return nothing

**Comment by joelworsham** [comment]
Same

**Comment by adaptifyDesigns** [comment]
Yes, similar, but slightly different, since the latest Gravity Forms release (1.9.2 and up), each name sub-field option has a toggle switch (Prefix, First, Middle, Last, Suffix). But regardless of which sub-field you toggle ON, all of them are displayed in the frontend... :-(

Has anyone found a better (more often updated) plugin that injects Foundation markup into the gravity forms output?

Thanks for any help.
